lib/entities/entry.ts

```diff
@@ -23,6 +24,12 @@ export type EntryProps<T = KeyValueMap> = {
 
 export type CreateEntryProps<TFields = KeyValueMap> = Omit<EntryProps<TFields>, 'sys'>
 
+export interface EntryReferenceProps<TObj> extends CollectionProp<TObj> {
+  includes?: {
+    [key: string]: TObj[]
```

Comment:
Not sure if we should just make it generic... but I think we _could_ try to type this a bit more:
from the references endpoint we know that it's going to be:

```
includes?: {
    Entry?: Collection<EntryProps>,
    Asset?: Collection<AssetProps>,
}
```